src/status_im/ui/components/checkbox/view.cljs

```diff
@@ -3,13 +3,16 @@
             [status-im.ui.components.react :as react]
             [status-im.utils.platform :as platform]))
 
-(defn- checkbox-generic [{:keys [on-value-change checked? accessibility-label style] :or {accessibility-label :checkbox}} plain?]
+(defn- checkbox-generic [{:keys [on-value-change checked? accessibility-label disabled? style] :or {accessibility-label :checkbox}} plain?]
   (let [icon-check-container (if plain? #() styles/icon-check-container)
         check-icon           (if plain? styles/plain-check-icon styles/check-icon)]
-    (if platform/android?
+    (if (or platform/android?
+            platform/desktop?)
       [react/view (merge styles/wrapper style)
        [react/check-box {:on-value-change     on-value-change
                          :value               checked?
+                         :disabled            (and (not checked?)
+                                                   disabled?)
                          :accessibility-label accessibility-label}]]
       [react/touchable-highlight (merge {:style               (merge styles/wrapper style)
```

Comment:
hmm, shouldn't we add this `disabled?` flag to iOS too?

Reply:
I will have a look actually :)